### PR TITLE
Update install-osx.md

### PR DESCRIPTION
**Pull request by umareddy**
@theopolis here are the changes required to install osqueryd on a new macbook using brew. Thought this would be useful and will save someone a few minutes.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by facebook-github-bot-6** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: docs/wiki/installation/install-osx.md [hunk 1]
```diff
@@ -43,6 +43,16 @@ To update osquery:
 $ brew update
 $ brew upgrade osquery
 ```
+Post installation steps to run osqueryd if you have never installed and run osqueryd on this Mac.
```
Comment (theopolis) [inline docs/wiki/installation/install-osx.md]: This sentence is a bit odd, is it meant to be a heading? Maybe: `#### Post installation steps` then the first sentence in the section can read: this only applies to the first time you're setting up `osqueryd` on this Mac.

File: docs/wiki/installation/install-osx.md [hunk 2]
```diff
@@ -43,6 +43,16 @@ To update osquery:
...
+
+After completing the brew installation run the following commands. If you are using the chef recipe to install osquery then these steps are not necessary, the recipe has this covered.
```
Comment (theopolis) [inline docs/wiki/installation/install-osx.md]: Maybe link to the chef recipe you're referencing, is this the chef we include in our docs? http://osquery.readthedocs.io/en/stable/deployment/configuration/#chef-os-x

**Pull request merged**